recipes/ecco_v4_py/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.1.6" %}
+
+package:
+  name: ecco_v4_py
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/e/ecco_v4_py/ecco_v4_py-{{ version }}.tar.gz
+  sha256: 419da92d36a739586f31c9b0bc132c1fd91b8cb33b5ef58064f64b2caa4a954b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
You can delete the `build` section here since `host takes care of everything you need.

Reply:
Thanks @synapticarbors! I'm a bit of a rookie to this so thanks for the review! `setuptools` does not need to be included in `host`?

Reply:
`pip` pulls in `setuptools`, so typically we don't include it.